roles/falcon_installation/README.md

```diff
@@ -57,7 +58,30 @@ This example installs the Falcon Sensor:
     falcon_client_id: <Falcon_UI_OAUTH_client_id>
     falcon_client_secret: <Falcon_UI_OAUTH_client_secret>
 ```
-
+---
+This example installs the Falcon Sensor at version N-2:
+```yaml
+---
+- hosts: all
+  roles:
+  - role: crowdstrike.falcon.falcon_installation
+  vars:
+    falcon_client_id: <Falcon_UI_OAUTH_client_id>
+    falcon_client_secret: <Falcon_UI_OAUTH_client_secret>
+    falcon_sensor_version: 2
```

Comment:
``falcon_sensor_version`` implies alignment to the actual crowdstrike sensor version (eg ``6.x.y``).

Should this be renamed to indicate sensor lag? Or perhaps update it to take a negative value (eg ``-2``) to indicate N-2?

Reply:
This is out-of-scope for this PR as `falcon_sensor_version` was merged in a previous PR. 

In general, I am not a fan of using a negative number here. The name could be updated but has already been deployed to other projects. Something we can think about for a follow on fix.